src/containers/Login/Login.js

```diff
@@ -62,7 +62,7 @@ export class Login extends Component {
         email: '',
         password: ''
       });
-      this.props.history.push('/');
+      this.props.history.push('/ratings');
```

Comment:
first one was correct. Redirect to ratings should be after user see all movies. There is should be button for redirect on all movies page